### PR TITLE
EPMLABSBRN-702: UI: add English description page

### PR DESCRIPTION
**Pull request by kateutlik**
[EPMLABSBRN-702](https://jira.epam.com/jira/browse/EPMLABSBRN-702)

**Description**:
- Added English transitions to the description page: - https://brainup.site/description
- Improved layout and readability

Used resources:
- translations: https://docs.google.com/document/d/1SK38DjZRL49YsyoSlBRaCIEs5FM6QHm93JKKlN79kB4/edit#heading=h.a0erdmxa65qd
- photos: https://docs.google.com/document/d/1QqgpH4aRRUu0fQNRBrN1rermmj99hLWLJIK4GZhaivg/edit?pli=1

Created a new document to track issues and user feedback in English:
https://docs.google.com/document/d/1ihOrk7RRmz92teGMsw068KifR5MWpoBo8wq2Lb7gL44/edit?usp=sharing

### CODE REVIEW

File: frontend/app/services/user-data.ts [hunk 1]
```diff
@@ -50,13 +50,11 @@ export default class UserDataService extends Service {
   }
 
   @action setLocale(localeName: string) {
-    const name = localeName === 'ru' ? 'ru-ru': 'en-us';
+    const name = localeName === 'ru' ? 'ru-ru' : 'en-us';
     this.intl.setLocale([name]);
     this.selectedLocale = name;
     localStorage.setItem('locale', name);
-    this.router.transitionTo('groups', { queryParams: { locale: name } });
```
Comment (kateutlik) [inline frontend/app/services/user-data.ts:57]: Removed redirect to login page after language change, so user stays on the page he is. Please check if anything else does not affected by this change.

**Review by kateutlik — COMMENTED** [review]
Currently there is duplicated content on the description page and the main page. You may check it by the following links:
- https://brainup.site/description
- https://brainup.site/
So we need to discuss this in the team and decide how we would like to fix it in the future.

As for the current story, please have a look at updated screens on the following screenshots:

- Russian language (updates are on the right hand)
image
image
image
image
image

- English version
image
image
image
image

**Review by ElenaSpb — APPROVED** [review]

File: frontend/app/services/user-data.ts (hunk 1 above)
Comment (lifeart) [reply frontend/app/services/user-data.ts:57]: we have different content for different languages (including id's), likely we have to figure out if/else logic here, to check current route

File: frontend/translations/en-us.yaml [hunk 2]
```diff
@@ -110,3 +110,111 @@ avatar_controls:
...
+  support_companies_header: With the support
+  our_people_header: Meet our team
+  our_people_thanks: We would like to thank everyone who participates and supports our project!
+  our_team_header: "We are grateful to all the contributors who have helped at different times and those who continue to develop the platform today."
+  team_member_image_alt: Team member photo {number}
+  our_medical_consultants: "And to our medical specialists, consultants."
+  doctor_image_alt: Doctor/consultant photo {number}
+  doctor1_name: Inna Koroleva
+  doctor1_details: "Academic supervisor of the rehabilitation department, Doctor of Psychology, Professor, author of the manuals \"Learning to listen and speak\""
+  doctor2_name: Ekaterina Garbaruk
```
Comment (lifeart) [inline frontend/translations/en-us.yaml:200]: let's have developers and doctors names/descriptions in json (or pojo), to improve maintainability of it

Comment (lifeart) [reply frontend/translations/en-us.yaml:200]: like 
```json
persons: {
  doctors: [
    { picture: '', lang: { 'ru-ru': { name, bio }, 'en-us': { name, bio } }
  ]
}

```

**Review by lifeart — APPROVED** [review]

**Review by lifeart — APPROVED** [review]

**Pull request merged**